northstar/src/runtime/mod.rs

```diff
@@ -56,8 +55,7 @@ pub(self) mod state;
 
 pub(self) type EventTx = mpsc::Sender<Event>;
 pub(self) type RepositoryId = String;
-pub use container::*;
-pub(self) use npk::manifest::{Name, Version};
+pub use api::container::*;
```

Comment:
Hm. I'm not sure whether this super central type `Container` should live in the `api` module. In general: there are some type that are used by `api`, `runtime` and even `npk`. 

Proposal: Create a `common` module that houses `Container`, `Notification` etc.. and is used by `runtime` `api` etc? This would avoid the need to have the `runtime` feature enabled when someone just uses `api` (means currently `nstar`?

What do you think?

Reply:
Currently `nstar`  uses only "api" from northstar. I don't see much trouble as it is currently. Maybe let us fix this first and then think about module organization later.

Reply:
Ok. Fine for me! I'll create a ticket!